src/utils/filename-utils.ts

```diff
@@ -18,7 +18,7 @@ export const getFileIndex = (dstPath: string, fileNamePrefix: string): number |
 
   const index = fs
     .readdirSync(dstPath)
-    .filter(file => file.indexOf(fileNamePrefix) > -1)
+    .filter(file => file.indexOf(fileNamePrefix+".") > -1) //with two note(Test2 and Test), the second note (Test) should not be renamed 
```

Comment:
I'm not sure on the real meaning of this, but I think the fileNamePrefix should be extended (if it's required where it is generated, not right before its usage)

Reply:
For example,  with Test2.md exist before hand, now with Test.md,  `file.indexOf(fileNamePrefix)` will return 0, as "Test" is included in "Test2.md", and resulted in "Test.2.md" but this case should not trigger auto rename.